### PR TITLE
Move draft requirements to future directions page

### PR DESCRIPTION
**Pull request by joshuagl**
Draft requirements which were removed from SLSA during the 1.0 cycle by un-defining Build L4 and removing the source track were left in the requirements and levels pages as an indicator of future direction.

However, their presence in the requirements page has been noted as a source of confusion, particularly for new readers.

In order to improve clarity of the requirements while maintaining open communication of potential future changes, move the draft requirements to a new "future directions" page.

Preview here: https://deploy-preview-616--slsa.netlify.app/spec/v1.0/future-directions

### CODE REVIEW

File: docs/spec/v1.0/future-directions.md [hunk 1]
```diff
@@ -0,0 +1,250 @@
+# Future directions
+
+The initial [draft version (v0.1)] of SLSA had a larger scope including
+protections against tampering with source code and a higher level of build
+integrity (Build L4). This page collects some early thoughts on how SLSA
+**might** evolve in future to re-introduce those notions and add other
```
Comment (di) [inline docs/spec/v1.0/future-directions.md:6]:
```suggestion
**might** evolve in future versions to re-introduce those notions and add other
```

File: docs/spec/v1.0/future-directions.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+
+</details>
+
+</section>
+
+<section id="source-track">
+
+## Source track
+
+A Source track could provide protection against tampering of the source code.
```
Comment (di) [inline docs/spec/v1.0/future-directions.md:120]:
```suggestion
A Source track could provide protection against tampering of the source code
prior to the build.
```

**Review by di — APPROVED** [review]
LGTM aside from nits. This resolves my concerns about including these with the rest of the spec!

File: docs/spec/v1.0/future-directions.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+-   Reproducible builds, which enable other systems to corroborate the
+    provenance.
+
+Draft requirements text for each of these requirements can be viewed below.
+Each entry might or might not be included in the future, in whole or in part.
+The list is not exhaustive.
+
+> WARNING: The draft requirements here are unversioned and subject to change.
+
+<details id="parameterless">
```
Comment (MarkLodato) [inline docs/spec/v1.0/future-directions.md:36]: Q: Do you think there's still value in copying these here, or should we just let readers click through the "Version 0.1" link to see the old text?

**Review by MarkLodato — APPROVED** [review]
Nice! Thanks!

File: docs/spec/v1.0/future-directions.md (hunk 3 above)
Comment (joshuagl) [reply docs/spec/v1.0/future-directions.md:36]: I agree that clicking through to the v0.1 requirements should suffice, I'll update the PR.

File: docs/spec/v1.0/future-directions.md (hunk 1 above)
Comment (joshuagl) [reply docs/spec/v1.0/future-directions.md]: Done, thanks.

File: docs/spec/v1.0/future-directions.md (hunk 2 above)
Comment (joshuagl) [reply docs/spec/v1.0/future-directions.md]: Good clarification, thanks.

**Review by mlieberman85 — APPROVED** [review]
LGTM

File: docs/spec/v1.0/future-directions.md (hunk 3 above)
Comment (joshuagl) [reply docs/spec/v1.0/future-directions.md]: Done.

**Pull request merged**